google/cloud/apigee/registry/applications/v1alpha1/registry_manifest.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryManifestProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// Manifest represents the dependency relation between resources, which 
+// is used by the controller to keep the resources up-to-date.
+message Manifest {
+  // List of dependency relations.
+  repeated Resource resources = 1 [
+		(google.api.field_behavior) = REQUIRED
+	];
+}
+
+message Resource {
+	string pattern = 1 [
+		(google.api.field_behavior) = REQUIRED
+	];
+	string filter = 2;
+	repeated SourcePattern dependsOn = 3;
```

Comment:
how about `repeated Dependency dependencies = 3;` ?

Reply:
(also FYI here's [style guidance](https://developers.google.com/protocol-buffers/docs/style#message_and_field_names) on message and field names)

Reply:
Thanks for that link. Renamed the field.